assets/js/googlesitekit/components-gm3/Checkbox.js

```diff
@@ -115,8 +102,8 @@ export default function Checkbox( {
 				<md-checkbox
 					id={ id }
 					ref={ ref }
-					role="checkbox"
```

Comment:
The `md-checkbox` now has its role automatically [set to `"production"`](https://github.com/material-components/material-web/blob/f9da93553bd64e7e8475f8acb8ee12206af12ac4/aria/delegate.ts#L52), overriding what we specify here, so there's no point setting it.